src/openzaak/utils/views.py

```diff
@@ -113,3 +128,162 @@ def initialize_request(self, request, *args, **kwargs):
         return super(viewsets.GenericViewSet, self).initialize_request(
             request, *args, **kwargs
         )
+
+
+class ImportCreateview(mixins.CreateModelMixin, viewsets.GenericViewSet):
+    import_type: ImportTypeChoices
+
+    permission_classes = (ImportAuthRequired,)
+
+    def get_queryset(self):
+        return Import.objects.all()
+
+    def create(self, request, *args, **kwargs):
+        existing_imports = self.get_queryset()
+        active_imports = existing_imports.filter(
+            import_type=self.import_type, status__in=ImportStatusChoices.started_choices
+        )
+
+        if active_imports:
+            raise ValidationError(
+                {
+                    "__all__": [
+                        _(
+                            "Er is een import process gaande. Probeer het later"
+                            " nogmaals."
+                        )
+                    ]
+                },
+                code="existing-import-started",
+            )
+
+        import_instance = Import.objects.create(
+            status=ImportStatusChoices.pending,
+            import_type=self.import_type,
+            # This should be updated when the import metadata file is parsed
+            total=0,
+        )
+
+        return Response(
+            data={
+                "upload_url": import_instance.get_upload_url(request=request),
+                "status_url": import_instance.get_status_url(request=request),
+                "report_url": import_instance.get_report_url(request=request),
+            },
```

Comment:
This can be added to a separate serializer, which can be used then by drf-spectacular, so there would be no need to specify it manually